independent-publisher-connectors/ExchangeRate/apiDefinition.swagger.json

```diff
@@ -0,0 +1,536 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Exchange Rate",
+    "description": "Get real-time currency conversion data provided by ExchangeRate-API.",
```

Comment:
@fordosa90 Please remove the word "API" from the description. Please follow our docs [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

Reply:
@Amjed-Ayoub It is the official name of the product/service: https://www.exchangerate-api.com/

Reply:
@fordosa90 Thank you so much for the confirmation. in that case yes can move forward with this.